File: ChatBot.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string userInput;

    while (true)
    {
        cout << "You: ";
        getline(cin, userInput);

        if (userInput == "exit")
        {
            cout << "Chatbot: Thank you! Have a nice day! " << endl;
            break;
        }
        else if (userInput == "hello")
        {
            cout << "Chatbot: Hello! How can I help you?" << endl;
        }
        else if (userInput == "hours")
        {
            cout << "Chatbot: Our shop is open from 9 AM to 9 PM." << endl;
        }
        else if (userInput == "services")
        {
            cout << "Chatbot: We provide home delivery and customer support." << endl;
        }
        else if (userInput == "contact")
        {
            cout << "Chatbot: You can call us at +123-456-7890." << endl;
        }
        else
        {
            cout << "Chatbot: Sorry, I didn't understand that." << endl;
        }
        
    }

    return 0;
}

File: ChatBot.py
def chatbot():
    print("Chatbot: Hello! Type 'exit' to end the chat.")
    print("Available commands: hello, hours, services, contact")

    while True:
        user_input = input("You: ").strip().lower()

        if user_input == "exit":
            print("Chatbot: Thank you! Have a nice day!")
            break
        elif user_input == "hello":
            print("Chatbot: Hello! How can I help you?")
        elif user_input == "hours":
            print("Chatbot: Our shop is open from 9 AM to 9 PM.")
        elif user_input == "services":
            print("Chatbot: We provide home delivery and customer support.")
        elif user_input == "contact":
            print("Chatbot: You can call us at +123-456-7890.")
        else:
            print("Chatbot: Sorry, I didn't understand that. Try: hello, hours, services, contact, exit.")

# Run the chatbot
chatbot()
